optuna/multi_objective/_hypervolume/base.py

```diff
@@ -4,11 +4,46 @@
 
 
 class BaseHypervolume(object, metaclass=abc.ABCMeta):
-    """Base class for hypervolume calculators"""
+    """Base class for hypervolume calculators
+
+        .. note::
+            In Optuna, this class is used for computing the hypervolume of objective points.
+            In other words, the coordinate of each point represents the one of the multi-objective
+            function's values.
```

Comment:
A mere suggestion but how about the following? I think the English is a bit clearer and shorter. I have not checked the line length.
```suggestion
            This class is used for computing the hypervolumes of points in multi-objective space.
            Each coordinate of each point represents one value of the multi-objective function.
```